helm/alfresco-content-services/README.md

```diff
@@ -97,10 +97,12 @@ Hence, setting up explicit Container memory and then assigning a percentage of i
 | email.server.enabled | bool | `false` | Enables the email server - see https://docs.alfresco.com/6.2/concepts/email-inboundsmtp-props.html |
 | filestore | object | `{"environment":{"JAVA_OPTS":" -XX:MinRAMPercentage=50 -XX:MaxRAMPercentage=80","scheduler.cleanup.interval":"86400000","scheduler.content.age.millis":"86400000"},"image":{"internalPort":8099,"pullPolicy":"IfNotPresent","repository":"quay.io/alfresco/alfresco-shared-file-store","tag":"0.16.1"},"initContainer":{"image":{"pullPolicy":"IfNotPresent","repository":"busybox","tag":"1.34.1"},"resources":{"limits":{"memory":"10Mi"},"requests":{"memory":"5Mi"}}},"livenessProbe":{"initialDelaySeconds":10,"livenessPercent":150,"livenessSavePeriodSeconds":600,"periodSeconds":20,"timeoutSeconds":10},"readinessProbe":{"initialDelaySeconds":20,"periodSeconds":60,"timeoutSeconds":10},"replicaCount":1,"resources":{"limits":{"memory":"1000Mi"},"requests":{"memory":"1000Mi"}},"service":{"externalPort":80,"name":"filestore","type":"ClusterIP"}}` | Declares the alfresco-shared-file-store used by the content repository and transform service |
 | global.ai | object | `{"enabled":false}` | Choose if you want AI capabilities (globally - including ADW AI plugin) |
-| global.alfrescoRegistryPullSecrets | string | `nil` | If there is a need to pull images from a private docker repo, a secret can be defined in helm and passed as an argument to the install command: e.g.: helm install alfresco-content-services --set global.alfrescoRegistryPullSecrets=quay-registry-secret or uncomment the following line if you don't want/need to pass it as a parameter on every install command : global.alfrescoRegistryPullSecrets: quay-registry-secret for more information: https://github.com/Alfresco/alfresco-anaxes-shipyard/blob/master/SECRETS.md -- Global definition of Docker registry pull secret which can be accessed by dependent ACS Helm chart(s) |
+| global.alfrescoRegistryPullSecrets | string | `nil` |  |
 | global.registryPullSecrets[0] | string | `"quay-registry-secret"` |  |
 | global.strategy.rollingUpdate.maxSurge | int | `1` |  |
 | global.strategy.rollingUpdate.maxUnavailable | int | `0` |  |
+| global.tracking.auth | string | `"secret"` | Select how solr and repo authenticate to each other none: work only prior to acs 7.2 (and was the default) secret: use a shared secret (to specify using `tracking.sharedsecret`) https: to use mTLS auth (require appropriate certificate configuration) |
+| global.tracking.sharedsecret | string | `"F7nRvqh0jKJnGlhzLKsdQiZ3/37x6Yb7f8N8i0HE52Q"` | Shared secret to authenticate repo/solr traffic |
```

Comment:
there are any minimum requirements for this secret?
suggesting to use something like `openssl rand -hex 20` may strength security